### PR TITLE
Feature/date range

### PR DESCRIPTION
**Pull request by ChristopherHarwell**

### CODE REVIEW

File: src/components/Layout/index.tsx [hunk 1]
```diff
@@ -27,7 +25,12 @@ export default function index({ children }) {
     <div>
       <Navbar />
       <main>{children}</main>
-      <Zendesk zendeskKey={process.env.REACT_APP_ZENDESK_KEY} {...setting} />
+      <div>
```
Comment (iesous-kurios) [inline src/components/Layout/index.tsx:28]: Not sure this is the best way, it shows up weird on my screen

Screen Shot 2021-04-15 at 7 51 00 PM

**Review by iesous-kurios — CHANGES_REQUESTED** [review]
Think we should separate the 'powered by family promise' from the date range feature so we can merge them separately

File: src/components/Layout/index.tsx (hunk 1 above)
Comment (ChristopherHarwell) [reply src/components/Layout/index.tsx:28]: Forgot about this but just fixed it.

**Review by iesous-kurios — APPROVED** [review]
None breaking, plus this isn't the production/dev site anymore, so no repercussions!

**Pull request merged**